recipes/moltenvk/all/conandata.yml

```diff
@@ -36,6 +36,11 @@ sources:
     url: "https://github.com/KhronosGroup/MoltenVK/archive/v1.1.0.tar.gz"
     sha256: "0538fa1c23ddae495c7f82ccd0db90790a90b7017a258ca7575fbae8021f3058"
 patches:
+  "1.2.2":
+    - patch_file: "patches/1.2.0-0001-fix-version-number-icd-json.patch"
```

Comment:
Looks like you repushed the old version of this change.

Reply:
Thank you for catching that, I had missed the commit which addressed this.  That is addressed now.

Reply:
No problem - thanks again for pushing this forward :)